admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -636,12 +636,20 @@ def _loadOneFile(self, conn, database, table, file, csvPrefix):
         """Load data from a single file into existing table"""
         cursor = conn.cursor()
 
-        # need to know field separator, default is the same as in partitioner.
-        separator = self.partOptions.get(csvPrefix + '.delimiter', '\t')
+        # need to know field delimiter, default is the same as in partitioner
+        special_chars = [self.partOptions.get(csvPrefix + '.delimiter', '\t'),
+                         self.partOptions.get(csvPrefix + '.enclose', '"'),
+                         self.partOptions.get(csvPrefix + '.escape', '\\'),
+                         self.partOptions.get(csvPrefix + '.newline', '\n')]
+
+        special_chars = map(repr, special_chars)
 
         self._log.info('load table %s from file %s', table, file)
-        q = "LOAD DATA LOCAL INFILE '%s' INTO TABLE %s.%s FIELDS TERMINATED BY '%s'" % \
-            (file, database, table, separator)
+        q = "LOAD DATA LOCAL INFILE '%s' INTO TABLE %s.%s" % (file,
+                                                              database,
+                                                              table)
+        q += (" FIELDS TERMINATED BY {0} ENCLOSED BY {1} ESCAPED BY {2}"
+              " LINES TERMINATED BY {3}").format(*special_chars)
```

Comment:
The order of the arguments is not obvious here, what happens if someone re-oders `special_chars` list.